docs/src/inputs/buttonGroup/examples/buttonGroupDefault.jsx

```diff
@@ -0,0 +1,22 @@
+import React, {
+    useState,
+} from 'react';
+import { ButtonGroup } from 'react-cm-ui';
+
+function ButtonGroupSample() {
```

Comment:
Change to `ButtonGroupDefault`.
If there are not really any variations for Button Group that need to be shown in the docs for right now, maybe get rid of this, and just put a big "Coming Soon" on the Dev Sandbox page (or get rid of Dev Sandbox page altogether if you'd prefer)

Reply:
I'd recommend to keep this structure for now so we'll be feel forced to add some other variations to the docs soon, because we already have several pages with the "Coming Soon" label, so I'd like to add them right after releasing the giving MVP (once I get some time to do it).
Thanks @groberts314 .

Reply:
Sounds good.  Thanks for doing the rename.

> once I get some time to do it

🤣 [Surely you're joking](https://en.wikipedia.org/wiki/Surely_You're_Joking%2C_Mr._Feynman!) Señor Mendoza!